src/components/menubar/menubar.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import IconButton from '@material-ui/core/IconButton';
+import Helmet from 'react-helmet';
+import PropTypes from 'prop-types';
+
+import { withStyles } from '@material-ui/core/styles';
+
+import acmhack from './acmhack.svg';
+import hackLogo from './hack-logo.png';
+import menubtn from './menu.svg';
+import ButtonBar from './buttonbar';
+import { Collapse } from '@material-ui/core';
+
+const StyleHeader = () =>
+	<Helmet>
+		<link href="https://fonts.googleapis.com/css?family=Poppins:400,500" rel="stylesheet"/>
```

Comment:
Let's put this somewhere more global than `menubar.js`, coalescing with https://github.com/uclaacm/hack.uclaacm.com/blob/a489eebba4bab8ec338fee8420cce7cb8ed721c1/src/components/homepage/homepagetemplate.js#L9-L11 in the process.

Reply:
Jody is working on homepagetemplate now, so this font dependency should be resolved in her PR.